packages/reactivity/src/reactive.ts

```diff
@@ -266,16 +266,20 @@ function createReactiveObject(
   ) {
     return target
   }
-  // target already has corresponding Proxy
-  const existingProxy = proxyMap.get(target)
-  if (existingProxy) {
-    return existingProxy
-  }
+
   // only specific value types can be observed.
   const targetType = getTargetType(target)
   if (targetType === TargetType.INVALID) {
     return target
   }
+
+  // only a whitelist of value types can be observed.
```

Comment:
I believe this line can be deleted.

I think something may have gone wrong with merging in `main`. This comment was reworded in #5332 and the new wording is a few lines up.

Reply:
done